server/src/main/java/org/candlepin/pinsetter/tasks/ExportJob.java

```diff
@@ -92,7 +92,7 @@ public void toExecute(JobExecutionContext context) throws JobExecutionException
     public static JobDetail scheduleExport(Consumer consumer, String cdnLabel, String webAppPrefix,
         String apiUrl, Map<String, String> extensionData) {
         JobDataMap map = new JobDataMap();
-        map.put(JobStatus.OWNER_ID, consumer.getOwner().getKey());
+        map.put(JobStatus.OWNER_ID, consumer.getOwnerId());
```

Comment:
This is a much needed sanity change, but without more changes in this class, this will break things. Gotta change the usage of the field to actual owner ID.